case_conversion/case_conversion.py

```diff
@@ -23,13 +23,13 @@ def camelcase(text, detect_acronyms=False, acronyms=[]):
     >>> camelcase("HELLO_HTML_WORLD", True, ["HTML"])
     'helloHTMLWorld'
     """
-    words, case, sep = case_parse.parse_case(text, detect_acronyms, acronyms)
+    words, _, _ = case_parse.parse_case(text, acronyms)
```

Comment:
nit: I like that you use underscores to indicate that these are unused, but can you keep the name as well so it's clear what is being unused:

```
words, _case, _sep = case_parse.parse_case(text, acronyms)
```

Reply:
Makes sense. (I was thinking one step ahead here.)